<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginSigninForm</class>
 <widget class="QWidget" name="LoginSigninForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>529</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>529</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>529</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Myanmar Text</family>
    <bold>true</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Login\Sign In</string>
  </property>
  <property name="windowOpacity">
   <double>11.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#LoginSigninForm {
	background-image: url(:/image/Login and sign up.jpg);

}
</string>
  </property>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>70</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>30</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 30pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEditL">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>221</y>
     <width>170</width>
     <height>28</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabelL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>221</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Username :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEditL">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>291</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabelL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>291</y>
     <width>111</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameLineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nationalCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>198</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameLineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ageLineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEditS">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SignInLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastNameLabel">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>190</y>
     <width>141</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Last Name : </string>
   </property>
  </widget>
  <widget class="QLabel" name="ageLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>71</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Age : </string>
   </property>
  </widget>
  <widget class="QLabel" name="firstNameLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>141</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>First Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nationalCodeLabel">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>240</y>
     <width>171</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>National Code : </string>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabelS">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>340</y>
     <width>121</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Username : </string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLableS">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>380</y>
     <width>141</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Password : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEditS">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="signinPushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
